beluga/test/beluga/sensor/data/test_sparse_value_grid.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright 2024 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(https://github.com/Ekumen-OS/beluga/issues/337): clang-format orders these includes differently depending on the
+// version.
+// clang-format off
+#include <beluga/sensor/data/sparse_value_grid.hpp>
+#include <Eigen/Core>
+#include <gmock/gmock.h>
+#include <gtest/gtest-typed-test.h>
+#include <gtest/gtest.h>
+#include <map>
+#include <optional>
+#include <unordered_map>
```

Comment:
Separating STL includes from others and listing them first might help make CI happy. See other test files.